src/Tizen.Multimedia.MediaPlayer/Player/PlayerError.cs

```diff
@@ -166,5 +170,21 @@ public ResourceLimitException(string message) : base(message)
         {
         }
     }
+
+    /// <summary>
+    /// The exception that is thrown when there it is not available.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class NotAvailableException : InvalidOperationException
```

Comment:
Does `NotAvailableException` means some kind of invalid operation?
If not, how about derive it from `Exception`?

Reply:
it is close to internal error. Exception might be better.